ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-presubmits.yaml

```diff
@@ -178,3 +178,59 @@ presubmits:
           - secret:
               name: cluster-secrets-azure
     trigger: ((?m)^/test upgrade,?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/conformance
+    decorate: true
+    labels:
+      artifacts: images
+    name: pull-ci-openshift-azure-conformance
+    rerun_command: /test conformance
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/e2e-azure-secret
+        - --template=/usr/local/e2e-azure-conformance
+        - --target=[output:stable:sync]
+        - --target=e2e-azure-conformance
+        command:
+        - ci-operator
+        env:
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: master.json
```

Comment:
This needs to be changed to `master.yaml`.

We have mass-renamed all ci-operator configuration files from `.json` to `.yaml` in https://github.com/openshift/release/pull/1336, because YAML is what ci-operator consumes now. JSON is a YAML subset so that the actual file content can stay like it is (we'll eventually mass-reformat them to the block style).